src/components/ProductPage/ProductMetaData.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import styled from 'styled-components'
+import numeral from 'numeral'
+import { InputProps } from 'react-neu'
+import { TokenDataProps } from './ProductDataUI'
+import IndexComponent from 'components/IndexComponent'
+import {
+  Bitcoin2xFlexibleLeverageIndex,
+  Ethereum2xFlexibleLeverageIndex,
+} from 'constants/productTokens'
+
+interface ProductMetaDataProps extends InputProps {
+  tokenData: TokenDataProps
+}
+
+const ProductMetaData: React.FC<ProductMetaDataProps> = ({ tokenData }) => {
+  const formatMetric = (metricValue: number) =>
+    numeral(metricValue).format('0.00a').toString().toUpperCase()
+  const formatLeverageMetric = (metricValue: number) =>
+    numeral(metricValue).format('0.00').toString().toUpperCase()
+  const formatDivergenceMetric = (metricValue: number) =>
+    numeral(Math.abs(metricValue)).format('0.00').toString().toUpperCase()
+  const formatSupplyCap = (metricValue: number | string) =>
+    numeral(parseInt(metricValue.toString()))
+      .format('0,0')
+      .toString()
+      .toUpperCase()
+
+  const isFLI =
+    tokenData.token.symbol === Ethereum2xFlexibleLeverageIndex.symbol ||
+    tokenData.token.symbol === Bitcoin2xFlexibleLeverageIndex.symbol
+
+  const realLeverage = tokenData.components
+    ? Number(tokenData.components[0].percentOfSet || 0) / 100
```

Comment:
I think I'm getting an error here:
`Cannot read property 'percentOfSet' of undefined`. Probably need optional chaining

Reply:
hmm... I didn't notice this in testing. I'll try to poke around